index.html

```diff
@@ -527,6 +527,13 @@ <h3>Proofs</h3>
 in Section <a href="#proof-chains"></a>.
           </dd>
 
+          <dt>nonce</dt>
+          <dd>
+An OPTIONAL string value supplied by the proof creator. An example use of this 
+field is to increase privacy by decreasing linkability to deterministically 
+generated signatures.
```

Comment:
I'm not sure `nonce` actually does what it's saying here when paired w/ a VC that has linkable values in it (like a deterministic signature, for example). Just noting that the reason for the property is not easily achieved unless you're using unlinkable stuff everywhere. 

This is not a blocking comment, just noting a concern with the current language. The language is true, but only in a certain context (that is not explained in the example).

Reply:
Hmm, this topic seems to be covered in section 6.1 on (Privacy Considerations/Unlinkability) where both data and signature linkability are discussed.